docs/querying/sql-functions.md

```diff
@@ -2473,27 +2575,89 @@ Shifts a timestamp forwards or backwards by a given number of time units.
 
 ## TIMESTAMP_TO_MILLIS
 
-`TIMESTAMP_TO_MILLIS(<TIMESTAMP>)`
+Returns the number of milliseconds since epoch for the given timestamp.
 
-**Function type:** [Scalar, date and time](sql-scalar.md#date-and-time-functions)
+* **Syntax:** `TIMESTAMP_TO_MILLIS(timestamp_expr)`
+* **Function type:** Scalar, date and time
 
-Returns the number of milliseconds since epoch for the given timestamp.
+<details><summary>Example</summary>
+
+The following example converts the `__time` column from the `taxi-trips` datasource into milliseconds since epoch.
+
+```sql
+SELECT 
+  "__time" AS "original_time",
+  TIMESTAMP_TO_MILLIS("__time") AS "miliseconds"
+FROM "taxi-trips"
+LIMIT 1
+```
+
+Returns the following:
+
+| `original_time` | `miliseconds` |
+| -- | -- |
+| `2013-08-01T08:14:37.000Z` | `1375344877000` |
+
+</details>
+
+[Learn more](sql-scalar.md#date-and-time-functions)
 
 ## TIMESTAMPADD
 
-`TIMESTAMPADD(<unit>, <count>, <TIMESTAMP>)`
+Adds a `unit` of time to `timestamp` a total of `count` times.
 
-**Function type:** [Scalar, date and time](sql-scalar.md#date-and-time-functions)
+* **Syntax:** `TIMESTAMPADD(unit, count, timestamp)`
+* **Function type:** Scalar, date and time
+
+<details><summary>Example</summary>
+
+The following example adds five months to the timestamp `2000-01-01 00:00:00`.
+
+```sql
+SELECT
+  TIMESTAMP '2000-01-01 00:00:00' AS "original_time",
+  TIMESTAMPADD (MONTH, 5, TIMESTAMP '2000-01-01 00:00:00') AS "new_time"
+```
+
+Returns the following:
+
+| `original_time` | `new_time` |
+| -- | -- |
+| `2000-01-01T00:00:00.000Z` | `2000-06-01T00:00:00.000Z` |
+
+</details>
 
-Adds a certain amount of time to a given timestamp.
+[Learn more](sql-scalar.md#date-and-time-functions)
 
 ## TIMESTAMPDIFF
 
-`TIMESTAMPDIFF(<unit>, <TIMESTAMP>, <TIMESTAMP>)`
+Returns the difference between two timestamps in the given `unit`.
```

Comment:
```suggestion
Returns the difference between two timestamps in a given unit.
```
Since timestamp is not in code font, let's remove code font from unit.